src/core/Akka.Persistence/Journal/AsyncWriteJournal.cs

```diff
@@ -253,43 +261,42 @@ private void HandleReplayMessages(ReplayMessages message)
                             // Send replayed messages and replay result to persistentActor directly. No need
                             // to resequence replayed messages relative to written and looped messages.
                             // not possible to use circuit breaker here
-                            ReplayMessagesAsync(context, message.PersistenceId, message.FromSequenceNr, toSequenceNr,
-                                message.Max, p =>
+                            ReplayMessagesAsync(context, message.PersistenceId, message.FromSequenceNr, toSequenceNr, message.Max, p =>
```

Comment:
If the `context` is being handled asynchronously, then we do need to close over it... however, the context is volatile - so I'm not sure why we'd want to snapshot it. What data are we using from it?